### PR TITLE
zh-trans: update content/zh/docs/admin/service-accounts-admin.md

### PR DESCRIPTION
**Pull request by SataQiu**
/cc @xichengliudui 
/assign @xichengliudui

### CODE REVIEW

File: content/zh/docs/admin/service-accounts-admin.md [hunk 1]
```diff
@@ -4,58 +4,58 @@ approvers:
...
   - 服务账户准入控制器（Service account admission controller）
-  - Token控制器（Token controller）
+  - Token 控制器（Token controller）
   - 服务账户控制器（Service account controller）
 
 ### 服务账户准入控制器
 
-对pod的改动通过一个被称为[Admission Controller](/docs/admin/admission-controllers)的插件来实现。它是apiserver的一部分。
-当pod被创建或更新时，它会同步地修改pod。 当该插件处于激活状态(在大多数发行版中都是默认的)，当pod被创建或更新时它会进行以下动作：
+对 pod 的改动通过一个被称为[Admission Controller](/docs/admin/admission-controllers) 的插件来实现。它是 apiserver 的一部分。
```
Comment (xichengliudui) [inline content/zh/docs/admin/service-accounts-admin.md]:
```suggestion
对 pod 的改动通过一个被称为 [Admission Controller](/docs/admin/admission-controllers) 的插件来实现。它是 apiserver 的一部分。
```

**Comment by SataQiu** [comment]
DONE

**Pull request merged**